src/src/PluginStructs/P164_data_struct.cpp

```diff
@@ -568,37 +638,27 @@ bool P164_data_struct::addCustomStep(uint16_t time, bool measureHP0, bool measur
 ///////////////////////////////////////////////////////////////////////////////////////////////////
 bool P164_data_struct::measureRaw() {
   uint8_t i2cbuf[8];
```

Comment:
`i2cbuf` is declared several times in this file, but never used.

Reply:
Left over from original Sciosoft version. Removed.